app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    should_be_vaccinated = False
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            should_be_vaccinated = True
```

Comment:
There is no need for `should_be_vaccinated` boolean flag, in case `VaccineError` is caught there should be a return statement with a required message.